client/package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
```

Comment:
It's a detail but maybe we should have something to distinguish scripts that build react from scripts that build docker

Reply:
agreed. done.